third_party/gif_decoder/gradle.properties

```diff
@@ -0,0 +1,11 @@
+POM_NAME=Glide Gif Decoder Library
+POM_ARTIFACT_ID=gifdecoder
+POM_PACKAGING=aar
+
+VERSION_NAME=1.0.0-SNAPSHOT
```

Comment:
@sjudd what about these version numbers?
(including the disklrucache in sjudd/disklrucache#3)